public/static/pathways/her2_pathway.json

```diff
@@ -0,0 +1,206 @@
+{
+  "name": "Early Stage HER2+ Pathway",
```

Comment:
could you also add "(Draft)" to the title? That was requested by our collaborators to try to take the focus off the details